airbase/pom.xml

```diff
@@ -166,7 +166,7 @@
         <dep.packaging.version>0.163</dep.packaging.version>
 
         <!-- Dependency versions that should be the same everywhere. -->
-        <dep.guice.version>4.0</dep.guice.version>
+        <dep.guice.version>4.1.0</dep.guice.version>
```

Comment:
Guice 4.2 is supposed to come out over the next few days.

Reply:
I'll update to that and put up a separate PR for just Guice